src/main/scala/uk/gov/homeoffice/drt/ports/config/Pik.scala

```diff
@@ -56,6 +56,7 @@ object Pik extends AirportConfigLike {
       )),
     flexedQueues = Set(),
     desksByTerminal = Map(T1 -> 5),
-    feedSources = Seq(ApiFeedSource, LiveBaseFeedSource, LiveFeedSource, AclFeedSource)
+    feedSources = Seq(ApiFeedSource, LiveBaseFeedSource, LiveFeedSource, AclFeedSource) ,
```

Comment:
I think we want to remove `LiveBaseFeedSource` and `AclFeedSource` from `feedSources` rather than add exemptions because these feeds are not used at all